source/cloud-native-platform/onboarding/team/slack.html.md.erb

```diff
@@ -38,3 +38,9 @@ Make sure you pick the Jenkins \[app] user:
 
 This can happen if your Slack invite was approved but you haven't been added to the SSO exclusion list.
 Get someone in your team to ask in the #pls-admins channel to check on your account.
+
+### I am getting an error message in my pipeline about `channel_not_found`
```

Comment:
Probably belongs on the troubleshooting page? this page already tells you to invite the bot.

Might be fine on here if people are searching though